cargo-dist/src/tasks.rs

```diff
@@ -1397,7 +1397,25 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
 
         // Try to lookup this version in the changelog
         let version_string = format!("{}", announcing_version);
-        let Some(release_notes) = changelogs.get(&*version_string) else {
+        let release_notes = if let Some(release_notes) = changelogs.get(&*version_string) {
+            release_notes
+        } else if !announcing_version.pre.is_empty() {
```

Comment:
ok actually after staring at this code i think this check should go *after* looking up if there are pre-release specific notes, if that makes sense?

Reply:
it does -- first we check the entire version-string blindly (1.0.0-prerelease) and then IF that fails AND we notice there was a suffix THEN we take that branch (checking bare 1.0.0).